movieReviewApp/src/main/java/com/mobiquity/movieReviewApp/controller/ContentController.java

```diff
@@ -0,0 +1,59 @@
+package com.mobiquity.movieReviewApp.controller;
+
+import com.mobiquity.movieReviewApp.Entity.Movie;
+import com.mobiquity.movieReviewApp.Entity.Series;
+import com.mobiquity.movieReviewApp.model.ResponseMovieApp;
+import com.mobiquity.movieReviewApp.service.MovieService;
+import com.mobiquity.movieReviewApp.service.SeriesService;
+import java.util.Arrays;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/content")
+public class ContentController {
+
+  private MovieService movieService;
+  private SeriesService seriesService;
+
+  public ContentController(MovieService movieService, SeriesService seriesService) {
+    this.movieService = movieService;
+    this.seriesService = seriesService;
+  }
+
+  @PostMapping("/addMovie")
+  public ResponseEntity<Object> addMovie(@RequestBody Movie movie) {
+    return new ResponseEntity<>(movieService.addMovie(movie), HttpStatus.OK);
+  }
+
+  @GetMapping("/movie")
+  public ResponseEntity<Object> getMovie(@RequestParam(name = "name") String name) {
+    Movie movie = movieService.getMovie(name);
+    if(movie != null){
```

Comment:
shouldn't be handled by the controller. should be handled by the service and return a Movie is okay, or throw exception if not okay, and the ControllerAdvice deals with the thrown exception

Reply:
same comment from me too. better handle in service.